components/graphs/IndicatorGraph.tsx

```diff
@@ -40,9 +40,12 @@ const createLayout = (
     yaxis: {
       automargin: true,
       hoverformat: ',.3r',
-      tickformat: ',.1r',
+      tickformat: yRange.ticksRounding
```

Comment:
Could `ticksRounding` be `0`? And if so, should this be:

```suggestion
      tickformat: typeof yRange.ticksRounding === 'number'
```

Reply:
Brilliant spot, thanks @woodwoerk !